swift/vapor-framework/Package.resolved

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
could we remove this file ? not required for a **benchmarking** purpose (we want the most up to date version)

Reply:
Sure